### PR TITLE
Bescka patch 2

### PR DESCRIPTION
**Pull request by bescka**
Patch Notes:
Features 
* Added blog concept styling/ favicon/ header and navbar/ placeholder text and navigation
* Added useful links section 
* Added first article

### CODE REVIEW

File: users/bescka/cuba chups.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+<body>
+  <div class="container">
+    <div class="ascii-art">
+      - ~- ~ - - O o <br>
+      &nbsp; ~- - ~ - o -<br>
+      ~ - ><(((º> ~ -<br>
+        &nbsp; - ~ - - -- ~ <br>
+    </div>
+    <span class="title_container">
```
Comment (maxitect) [inline users/bescka/cuba chups.html:21]: You could make the page header into a link back to the home page

File: users/bescka/floating_point_arithmetic_web.html [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+
+<body>
+    <div class="container">
+        <div class="ascii-art">
+            - ~- ~ - - O o <br>
+            &nbsp; ~- - ~ - o -<br>
+            ~ - &gt;&lt;(((º&gt; ~ -<br>
+            &nbsp; - ~ - - -- ~ <br>
+        </div>
+        <span class="title_container">
```
Comment (maxitect) [inline users/bescka/floating_point_arithmetic_web.html:23]: You could make the page header into a link back to the home page

**Review by maxitect — APPROVED** [review]
I realised that by commenting perhaps it falls on me to review and approve, I've not approved a PR before. Feel free to resolve all my comments and merge if this works, it looks great!

File: users/bescka/cuba chups.html (hunk 1 above)
Comment (bescka) [reply users/bescka/cuba chups.html]: Good idea! Implemented.

File: users/bescka/floating_point_arithmetic_web.html (hunk 2 above)
Comment (bescka) [reply users/bescka/floating_point_arithmetic_web.html]: as above

File: users/bescka/floating_point_arithmetic_web.html
[the remark this answers is not in the document]
Comment (bescka) [reply users/bescka/floating_point_arithmetic_web.html:1]: That's kind! Thanks :)

**Pull request merged**